src/login/tests/LoginPage.test.jsx

```diff
@@ -97,67 +97,108 @@ describe('LoginPage', () => {
 
   it('should submit form for valid input', () => {
     store.dispatch = jest.fn(store.dispatch);
-    const loginPage = mount(reduxWrapper(<IntlLoginPage {...props} />));
 
-    loginPage.find('input#emailOrUsername').simulate('change', { target: { value: 'test@example.com' } });
-    loginPage.find('input#password').simulate('change', { target: { value: 'password' } });
-    loginPage.find('button.btn-brand').simulate('click');
+    render(reduxWrapper(<IntlLoginPage {...props} />));
+
+    fireEvent.change(screen.getByText(
```

Comment:
It would be good if we use the relevant functions like we can use `getByRole()`
Please also follow in other places.